src/DSharp.Mscorlib/Threading/Tasks/TaskAwaiter.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Runtime.CompilerServices;
+
+namespace System.Threading.Tasks
+{
+    public sealed class TaskAwaiter : INotifyCompletion
```

Comment:
Where is the JS for this?

Reply:
It' only there so the C# code compiles with await... when it's compiled to js the native "promise" type is used so no taskawaiter exists... and if you target .net code it will use the one in the full mscorelib